data/json/npcs/scrap trader/scrap trader missions.json

```diff
@@ -0,0 +1,135 @@
+	"dialogue": {
+      "describe": "I need help...",
+	  "offer": "That machine I put together needs some repairs on it, but I'm out of soldering wire. If you could get me about 100 spools, I'll pay you for it.",
+	  "advice": "An electronics store might have some.",
+      "accepted": "I'll be here when you get back. Good luck!",
+      "rejected": "Someone's got to do it.",
+      "inquire": "Do you have the solder?",
+      "success": "Wonderful! This should do for a while.",
+      "success_lie": "Where are they? I don't believe you.",
+      "failure": "That's unfortunate. I guess I'll do it myself."
+    }
+  },
+{
+    "id": "MISSION_SCRAPPER_SET_TRADE_ROUTE",
+    "type": "mission_definition",
+    "name": "Establish A Trade Route",
+    "description": "Establish trade relations between the scrapper and a refugee center.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "trade_writ",
+	"count": 1,
+    "difficulty": 35,
+    "value": 10000,
+	"origins": [ "ORIGIN_OPENER_NPC" ],        
+    "followup": "MISSION_SCRAPPER_SET_TRADE_ROUTE_BLACKSMITH",    
+        "start": {
+                  "assign_mission_target": { "om_terrain": "evac_center_18", "reveal_radius": 0 }
+                 },
+	"has_generic_rewards": true,
+	"dialogue": {
+      "describe": "Go negotiate a trade agreement.",
+	  "offer": "I was out scavenging recently, and I found one of those old evac shelters. Turns out the computer was still on, and had an address for a refugee center. I figure they might want some spare metal, so I need you to go over there and strike a deal with them.",
+	  "advice": "It should be pretty simple making the deal, but I'd recommend taking a car over there myself.",
+      "accepted": "I'll be here when you get back. Good luck!",
+      "rejected": "Please? This is really important to me.",
+      "inquire": "What did they say about it?",
+      "success": "Thank you!",
+      "success_lie": "How does this help us?",
+      "failure": "That's unfortunate. I guess we'll have to find someone else to trade with."
+    }
+  },
+{
+    "id": "MISSION_SCRAPPER_SET_TRADE_ROUTE_BLACKSMITH",
+    "type": "mission_definition",
+    "name": "Establish A Trade Route",
+    "description": "Establish trade relations between the scrapper and some artisans.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "trade_writ",
+	"count": 1,
+    "difficulty": 35,
+    "value": 10000,
+	"origins": [ "ORIGIN_OPENER_NPC" ],  
+        "start": {
+                  "assign_mission_target": { "om_terrain": "isolated_house_farm_blacksmith", "reveal_radius": 0 }
+                 },
+	"has_generic_rewards": true,
+	"dialogue": {
+      "describe": "Go negotiate a trade agreement.",
+	  "offer": "Buisness had been booming since you made a deal with that refugee center. The last caravan that came by here mentioned something about a blacksmith down an old road. I think I might have another buisness opportunity, so I need you to go over there and make another deal.",

```

Comment:
```suggestion
	  "offer": "Business had been booming since you made a deal with that refugee center. The last caravan that came by here mentioned something about a blacksmith down an old road. I think I might have another business opportunity, so I need you to go over there and make another deal.",

```